packages/expo-video/ios/VideoViewManager.swift

```diff
@@ -0,0 +1,28 @@
+// Copyright 2024-present 650 Industries. All rights reserved.
+
+import AVKit
+import Foundation
+
+class VideoViewManager {
+  private var videoViews = Set<VideoView>()
```

Comment:
`Set` uses strong references, so it could potentially leak here if the view doesn't get unregistered. Consider using `NSHashTable<VideoView>.weakObjects()` instead.